### ISSUE TITLE
Support for page title being derived from omnibar

### ISSUE THREAD

**Issue by Blackbaud-AlexKingman** [blackbaud/skyux-sdk-builder#38]
@blackbaud-danbowman commented on [Thu Jul 27 2017](https://github.com/blackbaud/skyux2/issues/953)

final doc - https://blackbaudinc.box.com/s/ukcxs5hy3oqkjcr63iwlto1cx0e9wtd6 

To drive more consistent page titles naming, it'd be nice if SKY UX Builder had an option for a page's default title to derive from...

**Comment by Blackbaud-AlexKingman** [comment]
Closing in favor of blackbaud/skyux-core#67
